### ISSUE TITLE
The code component does not respect min_width

### ISSUE THREAD

**Issue by edmcman** [gradio-app/gradio#9414]
### Describe the bug

The code component ignores min_width

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr

with gr.Blocks() as demo:
    with gr.Row() as row:
        c1 = gr.Code(min_width=1000)
        c2 = gr.Code(min_width=1000)
        c3 = gr.Code(min_width=1000)
        c4 = gr.Code(min_width=1000)
        t = gr.Textbox(min_width=1000)

demo.queue()
demo.launch(server_name="0.0.0.0", server_port=7860)
```

### Screenshot

![image](https://github.com/user-attachments/assets/783db664-599e-44f7-849e-678d2d81a5ec)

### Logs

```shell
NA
```

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Linux
gradio version: 4.44.0
gradio_client version: 1.3.0

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 23.2.1
anyio: 4.4.0
fastapi: 0.111.0
ffmpy: 0.3.2
gradio-client==1.3.0 is not installed.
httpx: 0.27.0
huggingface-hub: 0.23.4
importlib-resources: 6.4.0
jinja2: 3.1.4
markupsafe: 2.1.5
matplotlib: 3.9.1
numpy: 1.26.4
orjson: 3.10.6
packaging: 24.1
pandas: 2.2.2
pillow: 10.4.0
pydantic: 2.8.2
pydub: 0.25.1
python-multipart: 0.0.9
pyyaml: 6.0.1
ruff: 0.5.1
semantic-version: 2.10.0
tomlkit==0.12.0 is not installed.
typer: 0.12.3
typing-extensions: 4.12.2
urllib3: 2.2.2
uvicorn: 0.30.1
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2024.5.0
httpx: 0.27.0
huggingface-hub: 0.23.4
packaging: 24.1
typing-extensions: 4.12.2
websockets: 11.0.3
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Looks like this is fixed on the `5.0-dev` branch via https://github.com/gradio-app/gradio/pull/9450, thanks @hannahblair!

**Comment by edmcman** [comment]
Great, thanks!
